### DISCUSSION TITLE
Embed pdf with pdf.js

### DISCUSSION THREAD

**Discussion by erwindeclerck** [Budibase/budibase#6909] [Help] [answered]
Hi,
I'm evaluating Budibase for a number of possible applications. Embedding pdfs the pdf.js way is imperative...
Is there a way to do this with Budibase?
Kind regards,
Erwin

**Comment by melohagan** [comment, marked as answer]
Hi @erwindeclerck 

At the moment there is no PDF component, but you can have PDF attachments. When you click on an attachment it would be opened in a new tab which might suffice? 
Seems a minor issue was actually fixed around this recently: https://github.com/Budibase/budibase/issues/6896

Otherwise with custom components being worked on this quarter, I'm sure this could be built in the future!
